book/src/getting-started/2-our-first-cxx-qt-module.md

```diff
@@ -7,21 +7,30 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # Our first CXX-Qt module
 
-As with all things Rust, we'll first want to create a cargo project.
+We first need to create a folder structure to add the different parts of our project.
+
+```ignore
+tutorial
+  - cpp
+  - qml
+  - rust
+```
+
+As with all things Rust, we'll want to create a cargo project, run the following command inside the `tutorial` folder to initialise the Rust part of the project.
```

Comment:
```suggestion
As with all things Rust, we'll want to create a cargo project, run the following command inside the `tutorial/rust` folder to initialise the Rust part of the project.
```

Reply:
If there's an empty `rust` directory when running `cargo init --lib rust` inside the `tutorial` directory, `cargo init` fails:
```
~/tutorial 
❯ cargo new --lib rust
error: destination `/home/be/tutorial/rust` already exists

Use `cargo init` to initialize the directory
```

Reply:
`cargo init --lib rust` works on an empty directory, `cargo new --lib rust` fails :-)

Reply:
Oh, I didn't realize there was a difference. Today I learned!